pkg/driver/node/regionprovider/imds.go

```diff
@@ -0,0 +1,34 @@
+package regionprovider
```

Comment:
Has any code here changed? The removed code isn't in this commit

Reply:
There is no code change (other than function name) in this package (`regionprovider/{imds, provider, provider_test}.go`). It just moved out of now removed `pkg/driver/node/mounter/credential_provider.go`